topics/whats-new-in-kotlin-for-kmm.md

```diff
@@ -8,6 +8,25 @@ Android Studio will recommend an automatic update to a new Kotlin release. You c
 
 Here you can find a short summary of the features Kotlin provides for developing multiplatform mobile applications.
 
+## Kotlin 1.5.20 for KMM
+
+[Kotlin 1.5.20](https://kotlinlang.org/docs/whatsnew1520.html) introduces a number of improvements and features that are helpful for KMM:
+
+* **Opt-in export of KDoc comments to generated Objective-C headers.**
+  You can now set the Kotlin/Native compiler to export the [documentation comments (KDoc)](https://kotlinlang.org/docs/kotlin-doc.html) from Kotlin code
+  to the Objective-C frameworks generated from it, making them visible to the frameworks’ consumers.
+  
+  This feature is experimental. We would appreciate your feedback on it in [YouTrack](https://youtrack.jetbrains.com/issue/KT-38600).
+
+  Learn more about [exporting of KDoc comments to generated Objective-C headers](https://kotlinlang.org/docs/whatsnew1520.html#opt-in-export-of-kdoc-comments-to-generated-objective-c-headers).
```

Comment:
I'd highlight the fact that you need to opt-in to use this feature. Maybe smth like 

Learn more about [exporting of KDoc comments to generated Objective-C headers] and how to opt-in?

Reply:
Done, thanks!